app/main.py

```diff
@@ -1 +1,53 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: object = True
```

Comment:
The type annotation for `is_hungry` should be `bool` instead of `object`. This will ensure that the type is consistent with its usage and the derived classes.